be/Cf.WebApi/Utilities/ServicesConfiguration/ServiceCollectionExtensions.cs

```diff
@@ -81,6 +81,18 @@ public static void ConfigureServices(this IServiceCollection services, IConfigur
 
             c.OperationFilter<SecurityRequirementsOperationFilter>();
         });
+
+        var MyAllowSpecificOrigins = "_myAllowSpecificOrigins";
+        services.AddCors(options =>
+        {
+            options.AddPolicy(name: MyAllowSpecificOrigins,
+                              builder =>
+                              {
+                                  builder.WithOrigins("https://localhost:3000")

```

Comment:
Im not sure if it is very important but you could also move this value to appsettings as well.
Optional:P